#156881784 Implement leaderboard

nzediegwu1 opened the pull request:
#### What does this PR do?
Implement leaderboard

#### Description of Task to be completed?
1. Create route for leader board
2. Implement method to get users sorted by points in descending order

#### How should this be manually tested?
On terminal, run command:
1. `npm run start`
In postman, 
2. Login and set `x-access-token` in headers
3. Send request to GET /api/leaderboard

#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
[#156881784](https://www.pivotaltracker.com/story/show/156881784)

#### Screenshots (if appropriate)
N/A

Commit:
[Feature #156881784] Implement leaderboard

1. Create route for leader board
2. Implement method to get users sorted by points in descending order

[Finishes #156881784]

d3mola reviewed (commented):
Good job Anaeze, your code is self-explanatory and easy to follow. However there's a typo on line 359, `nullify` should be `nulify`. Also consider adding a `@description` field to your `jsdoc` comment as that makes your already well written comment even better.

nzediegwu1 commented:
Thanks bro for the feedback on `@description` tag @d3mola , will include asap. On the other-hand, I think `Nullify` is the correct spelling.

Commit:
[Feature #156881784] Implement leaderboard

1. Create route for leader board
2. Implement method to get users sorted by points in descending order

[Finishes #156881784]

Commit:
Add description to leaderboard jsDoc

nzediegwu1 committed:
Merge branch 'develop' into ft-implement-leaderboard-156881784
